src/openapi/streaming/connection/constants.ts

```diff
@@ -1,55 +1,54 @@
 /**
  * Event that occurs when the connection state changes.
  */
-export const EVENT_CONNECTION_STATE_CHANGED = 'connectionStateChanged';
+export const EVENT_CONNECTION_STATE_CHANGED = 'connectionStateChanged' as const;

```

Comment:
I thought you only need as const on a object property, not on a const export

Reply:
This was added since TS converts consts assigned to a class property as a string. I realized that checking it in [this playground ](https://www.typescriptlang.org/play?#code/MYewdgzgLgBAwgeQHJIKJwCoElkH0DKGAghqrlkltkQDJYBaFA4jALwwDk0ATgKYCGAWwCWYAOYARYRFBgwvYFABKvAI4BXXtF4ATDgG4AsACgTAegBUFkzAswAsvzD8xWmBHUAjGd2EAHKGFwCBgoEFCAC14YBD9eMBgiAAUsdyg+IVExd15uADdhYF4AOhs7BDAimGAMqF1I6RgAd2EAG1aYYUFBXWF+OtaATxh+usEA0PDofm5YKCi0jJFxHPzC3jKzE2BW-ggQ-HSBZeyAbxsYS7MzGAydcCH4ZDRMHCQCYlJySmo6RiQWOxECh0Ng8IQSGQKFQsLQGMwjMZLpdgS8we8IV9ob94QC2E8Qa9wZ8oT9YX8ETBrjAADx0mAAWiZzJZzOaUQSTWiewgwjECX41WCUAANDAMPgYIJ+ABrNzCWCCnhZEwXS7pPmubgACmcPQAXJweMcslIZOB5IoVBotHU9DAAD6cABmIBAHAAlDBzqYkciAL6qv3q23ar0+5GRhoQYoasRa7XzaTFVGgt4fSHfGFw-5MD2IyOB4xFoPUgBqkqDzvUlUC4FCvnjuV1Ql4hq4R0y4jNsktyjUmm09qdHFd7vDQZLxjjCdTRIxJKzONz+aAA). An alternative would be to remove `as const` and add `readonly` modifier to each constant class property.

Reply:
I see so it is to avoid `as const` where we use them in the class?

Reply:
ok, looked at the playground. Weird!